packages/fx-core/src/component/resource/botService/botService.ts

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/**
+ * @author zhijie <zhihuan@microsoft.com>
```

Comment:
Although this file is created by Zhijie at first, you can update author to you as you are the owner of it.

Reply:
This is auto-injected by eslint. Will do.